app/Http/Controllers/ProjectController.php

```diff
@@ -106,7 +106,7 @@ public function update(Request $request, Project $project)
             'short_description' => $request->short_description,
             'long_description'  => $request->long_description,
             'github_link'       => $request->github_link,
-            'website_link'      => $request->website_link,
+            'siteweb_link'      => $request->siteweb_link,
```

Comment:
Okay so I will change to website_link in views and controller and change the siteweb_link field in database!